src/v3/src/util/isLtrField.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2024-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+// fields with these names should stay LTR by default
+const ltrOnlyFieldNames = [
+  'credentials.passcode',
+  'identifier',
+  'credentials.newPassword',
+  'confirmPassword',
+  'extension',
+];
+
+export const isLtrField = (fieldName: string) => ltrOnlyFieldNames.includes(fieldName) || fieldName.endsWith('phoneNumber');
```

Comment:
Extends the logic used in https://github.com/okta/okta-signin-widget/blob/master/src/v3/src/transformer/uischema/setLtrFields.ts to also include phone number fields since the `PhoneAuthenticator` is also always LTR. This utility is then used in `theme.ts` to add MUI theme overrides for LTR exceptions